beetmoverscript/src/beetmoverscript/utils.py

```diff
@@ -178,30 +173,6 @@ def generate_beetmover_template_args(context):
     return tmpl_args
 
 
-def _generate_beetmover_template_args_maven(task, release_props):
```

Comment:
This LGTM! the only place where we don't have declarative artifacts is handling the partner repacks. Everywhere else, we no longer rely on manifests, but on payload definitions so removing this here makes total sense as it's dead code.